_aggregations/metric/matrix-stats.md

```diff
@@ -9,8 +9,23 @@ redirect_from:
 
 # Matrix stats aggregations
 
-The `matrix_stats` aggregation generates advanced stats for multiple fields in a matrix form.
-The following example returns advanced stats in a matrix form for the `taxful_total_price` and `products.base_price` fields:
+The `matrix_stats` metric is a multi-value metric that generates covariance statistics for two or more fields in matrix form. 
+
+The `matrix_stats` metric does not support scripting.
```

Comment:
aggregation does not support scripting?

Reply:
@jainankitk That's my understanding. Please verify.

Reply:
Okay, my point was that the below reads slightly better?

```
The `matrix_stats` aggregation does not support scripting.
```

Reply:
```suggestion
The `matrix_stats` aggregation does not support scripting.
```